userguide/recommender/custom-model-ios.md

```diff
@@ -0,0 +1,50 @@
+# Custom Recommender Model for iOS Applications
+
+The Turi Create Recommender Model is also available for use in your iOS/macOS
+Applications as a custom model via exporting to Core ML. After creating your model in
+Turi Create, you need to export your model to Core ML by leveraging the simple 
```

Comment:
Simplify the wording here. "After creating your model in Turi Create, save it in Core ML format by calling `export_coreml` as follows".